### ISSUE TITLE
Throttling 

### ISSUE THREAD

**Issue by mjdavies** [nginx-proxy/nginx-proxy#761]
Hello there

This isn't an issue, just a question.

I need to enable some sort of throttling on all the virtual hosts, something along these lines

http://nginx.org/en/docs/http/ngx_http_limit_req_module.html

How do I alter the the base http settings?

Apologies if it's in the docs, I have read them but not really sure how to proceed.

Thanks

Matt

**Comment by thomasleveil** [comment]
The relevant part of the doc is [§ Custom Nginx Configuration](https://github.com/jwilder/nginx-proxy/blob/master/README.md#custom-nginx-configuration). Since you want your custom directive to apply to all virtual host, refer more specifically to [§ Proxy-wide](https://github.com/jwilder/nginx-proxy/blob/master/README.md#proxy-wide).

## tl;dr

- create a file with your custom nginx directives named `my_throttling.conf`
- mount that file in the nginx-proxy container with a volume: `docker run -v /somewhere/my_throtthing.conf:/etc/nginx/conf.d/my_throttling.conf ....`

**Comment by mjdavies** [comment]
Thanks @thomasleveil 

I'll give that a bash.

**Comment by elico** [comment]
Any recommedations?
